tests/foreman/cli/test_repository.py

```diff
@@ -361,8 +523,15 @@ def test_positive_update_url(self):
         @Assert: Repository url is updated
         """
         new_repo = self._make_repository()
+        # generate a tuple of auth repos with all valid credentials
+        auth_repos = ()
```

Comment:
`http://foo.bar/baz` and `http://monty:python@foo.bar/baz` seems as little bit different things, but if they are simply treated as strings and there are no special logic inside of Sat application then yes, we can proceed with your approach, but if we have any validation (that in result can fail that specific case) then we definitely should separate cases on two